app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/FavoritesScreen.kt

```diff
@@ -86,11 +86,9 @@ fun LazyGridScope.header(
 @Preview
 @Composable
 fun FavoritesScreenPreview() {
-    val favoritesMapper = FavoritesMapperImpl()
-    MovieAppTheme {
-        FavoritesScreen(
-            favoritesViewState = favoritesMapper.toFavoritesViewState(MoviesMock.getMoviesList()),
-            onNavigateToMovieDetails = {}
-        )
+    runBlocking {
+        FakeMovieRepository(Dispatchers.IO).addMovieToFavorites(2)
     }
+        MovieAppTheme {
+        }
```

Comment:
This is just an empty theme composable, remove it. If you want to create a screen preview, remove `runBlocking` and just add `FavoritesScreen` to preview with mocked `FavoritesViewState`. This applies to all screen previews